libcxx/docs/ReleaseNotes/18.rst

```diff
@@ -68,6 +69,13 @@ Improvements and New Features
   on a per translation unit basis using the ``_LIBCPP_HARDENING_MODE`` macro. See :ref:`the hardening documentation
   <using-hardening-modes>` for more details.
 
+- The ``_LIBCPP_ENABLE_CXX26_REMOVED_FEATURES`` macro has been added to allow
```

Comment:
I think we should avoid providing `_LIBCPP_ENABLE_CXX26_REMOVED_FEATURES` entirely. We should not make it *too* easy for folks to stay in a deprecated situation -- forcing them to select which features they actually need seems like a reasonable and desirable thing to me.

Reply:
In general I think that would be a nice new policy. For example, `std::string::reserve()` has a better alternative for a long time. I will adapt my patch for that paper to not add an `ENABLE` macro

For this feature there is no standard alternative. Zach Laine has written a paper with a replacement feature. AFAIK that paper is still on-track for C++26. Hard removing this feature would break users and make it harder to transition from C++23 to C++26. So I rather remove backwards compatibility for this case in C++29; assuming Zach's paper will be in C++26 and we've implemented that paper.

Reply:
To be clear, I am not suggesting that we make it impossible to re-enable the feature. I'm just saying that we could provide `_LIBCPP_ENABLE_CXX26_REMOVED_CODECVT` without providing `_LIBCPP_ENABLE_CXX26_REMOVED_FEATURES`.

Reply:
As discussed yesterday that seems fine to me.